test/org/zalando/stups/friboo/system/cron_test.clj

```diff
@@ -0,0 +1,16 @@
+(ns org.zalando.stups.friboo.system.cron-test
+  (:require [org.zalando.stups.friboo.system.cron :refer :all]
+            [clojure.test :refer :all]
+            [overtone.at-at :as at]))
+
+(def-cron-component TestCron [state]
+  (at/at (at/now) (job deliver state 42) pool))
+
+(deftest test-cron-component-lifecycle
+  ;; Here we make sure that the component is started and stopped properly.
+  (let [state (promise)
+        cron-component (map->TestCron {:state state})]
+    (.start cron-component)
```

Comment:
Good point, will change it. Actually, the test was enough to reproduce the crash, but you're right in saying that it doesn't actually stop the running component.